spec/chromedriver_updater_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe Geordi::ChromedriverUpdater do
+  describe '#run' do
+    context 'exit_on_failure=true (default)' do
+      it 'exits the command with status code 1 if an error occurs' do
+        allow(subject).to receive(:determine_chrome_version).and_raise(Geordi::ChromedriverUpdater::ProcessingError, 'Could not determine the version of Google Chrome.')
+        expect(Geordi::Interaction).to receive(:fail).with('Could not determine the version of Google Chrome.')
+
+        subject.run({})
+      end
+    end
+
+    context 'exit_on_failure=false' do
+      it 'prints an interaction note if an error occurs' do
+        allow(subject).to receive(:determine_chrome_version).and_raise(Geordi::ChromedriverUpdater::ProcessingError, 'Could not determine the version of Google Chrome.')
```

Comment:
When stubbing, I like to use custom return values to make the connection between stub and expectation clear. E.g.
```suggestion
        allow(subject).to receive(:determine_chrome_version).and_raise(Geordi::ChromedriverUpdater::ProcessingError, 'The error message')
```
Above as well.